### PR TITLE
docs(agents): name both references in the restore note

### PR DESCRIPTION
**Pull request by vanyastaff**
Follow-up to #615, addressing the review comment there: the new paragraph spoke of "references" in the plural and "citing one", but gave restore steps only for `.flutter/`. Both directories are gitignored local clones and either can be absent, so this names them explicitly — with the concrete command for `.flutter/` (the one actually used and verified) and a pointer for `.gpui/` rather than an untested invented command.

🤖 Generated with [Claude Code](https://claude.com/claude-code)

https://claude.ai/code/session_01Du3J3bDcsRaSU3tKgYKvni

### CODE REVIEW

File: AGENTS.md [hunk 1]
```diff
@@ -140,14 +140,16 @@ When changing render-tree, sliver, layout, paint, hit-test, semantics, schedulin
 
 **Read the reference for *what* and *why*, then write Rust from that understanding — do not transcribe.** Loyalty is to observable behavior (output, edge cases, ordering), not to Dart's structure, naming, or file layout. Confirm the match before reporting done — see [Definition of Done](#definition-of-done-anti-cheating).
 
-**The references are gitignored local clones, so they can be absent — check before citing one.** `ls .flutter` costs nothing and a missing reference has already produced hollow "verified against Flutter" claims here. Restore with a sparse shallow clone (~62 MB):
+**Both references are gitignored local clones, so either can be absent — check before citing one.** `ls .flutter` costs nothing and a missing reference has already produced hollow "verified against Flutter" claims here. Restore `.flutter/` with a sparse shallow clone (~62 MB):
```
[the remark this answers is not in the document]
Comment (vanyastaff) [reply AGENTS.md:143]: Technically right, and deliberately not spinning a third PR for it. The sentence already says both directories are local clones and that either can be absent, with "check before citing one" as the instruction; the `ls .flutter` is an example of the check, not an exhaustive list, and `.gpui/` is consulted rarely enough that the reader who needs it will substitute the path. Folding the extra mention into the next edit that touches this section rather than spending a CI cycle on one clause — flagging the decision here so it is a choice on the record, not an oversight.
